gatsby-config.js

```diff
@@ -10,138 +10,147 @@ const {
+      gatsbyRemarkPlugins: [
+        {
+          resolve: 'gatsby-remark-images',
+          options: {
+            maxWidth: 600,
           },
-          'gatsby-remark-responsive-iframe',
-          'gatsby-remark-smartypants',
-        ],
-        plugins: [
-          {
-            resolve: `gatsby-remark-images`,
-            options: {
-              maxWidth: 600,
-            },
+        },
+        'gatsby-remark-responsive-iframe',
+        'gatsby-remark-smartypants',
+      ],
+      plugins: [
+        {
+          resolve: 'gatsby-remark-images',
+          options: {
+            maxWidth: 600,
           },
-          'gatsby-remark-responsive-iframe',
-          'gatsby-remark-smartypants',
-        ],
-      },
+        },
+        'gatsby-remark-responsive-iframe',
+        'gatsby-remark-smartypants',
+      ],
     },
-    {
-      resolve: `gatsby-plugin-modal-routing`,
-      options: {
-        modalProps: {
-          style: {
-            content: {
-              left: 0,
-              right: 0,
-              top: 0,
-              bottom: 0,
-            },
+  },
+  'gatsby-plugin-modal-routing-3': {
+    resolve: 'gatsby-plugin-modal-routing-3',
+    options: {
+      modalProps: {
+        style: {
+          content: {
+            left: 0,
+            right: 0,
+            top: 0,
+            bottom: 0,
           },
         },
       },
     },
-    {
-      resolve: `gatsby-source-filesystem`,
-      options: {
-        name: `images`,
-        path: path.join(__dirname, `src`, `images`),
-      },
-    },
-    {
-      resolve: `gatsby-source-contentful`,
-      options: {
-        spaceId: CONTENTFUL_SPACE,
-        accessToken: CONTENTFUL_TOKEN,
-        environment: 'master',
-      },
-    },
-    {
-      resolve: 'gatsby-source-lever',
-      options: {
-        site: 'yld',
-        verboseOutput: false,
-      },
+  },
+  'gatsby-source-filesystem': {
+    resolve: 'gatsby-source-filesystem',
+    options: {
+      name: 'images',
+      path: path.join(__dirname, 'src', 'images'),
     },
-    {
-      resolve: `gatsby-plugin-google-gtag`,
-      options: {
-        trackingIds: [GA_TRACKING_ID],
-        gtagConfig: {
-          anonymize_ip: false,
-          cookie_expires: 0,
-        },
-        pluginConfig: {
-          head: true,
-        },
-      },
+  },
+  'gatsby-source-contentful': {
+    resolve: 'gatsby-source-contentful',
+    options: {
+      spaceId: CONTENTFUL_SPACE,
+      accessToken: CONTENTFUL_TOKEN,
+      environment: 'master',
     },
-    {
-      resolve: `gatsby-plugin-favicon`,
-      options: {
-        logo: './src/images/favicon.png',
-        appName: 'YLD',
-        background: '#fff',
-      },
+  },
+  'gatsby-source-lever': {
+    resolve: 'gatsby-source-lever',
+    options: {
+      site: 'yld',
+      verboseOutput: false,
     },
-  ],
+  },
+};
+
+module.exports = {
+  siteMetadata: {
+    siteTitle: 'YLD',
```

Comment:
According to the Gatsby docs, I think this should be title rather than siteTitle
```suggestion
    title: 'YLD',
```